.github/workflows/build-jsp.yml

```diff
@@ -43,16 +43,16 @@ jobs:
       run: >
         mkdir -pv $(pwd)/output/Generated/ &&
         mvn -s .github/workflows/mvn-settings.xml -B exec:java
-        -Dexec.mainClass="org.unicode.text.UCD.Main" -Dexec.args="version 15.0.0 build MakeUnicodeFiles"
+        -Dexec.mainClass="org.unicode.text.UCD.Main" -Dexec.args="version 15.1.0 build MakeUnicodeFiles"
```

Comment:
Is there any way to expand this from the `CURRENT_UVERSION` token in cli-build-instructions.yml?

Reply:
I *suspect* not, since it's in a different workflow file. But I am no expert in GitHub Actions.

Reply:
I am _entirely_ unsure myself, but even if we needed to create a new file containing just the constant such that it could be used by the others I think it would be worth doing.

Reply:
--> #344